coreax/util.py

```diff
@@ -201,27 +202,39 @@ def solve_qp(kernel_mm: ArrayLike, kernel_matrix_row_sum_mean: ArrayLike) -> Arr
     return sol.primal
 
 
-def jit_test(fn: Callable, *args, **kwargs) -> tuple[float, float]:
+def jit_test(
```

Comment:
We should add another unit test in tests/unit/test_util.py that considers non-empty/None values of the new input arguments, to verify they are handled correctly.

Reply:
Will do. Just to note. these cases are already implicitly tested in the performance tests.